eo-phi-normalizer/test/eo/phi/rules/yegor.yaml

```diff
@@ -120,12 +120,12 @@ rules:
   - name: R_COPY
     description: 'Application of α-binding'
     pattern: |
-      ⟦ !τ ↦ ∅, !B ⟧(!τ ↦ !b)
+      ⟦ !τ1 ↦ ⟦ !τ2 ↦ ∅, !B1 ⟧(!τ2 ↦ !n, !B2), !B3 ⟧
     result: |
-      ⟦ !τ ↦ !b, !B ⟧
+      ⟦ !τ1 ↦ ⟦ !τ2 ↦ !n[ ξ ↦ ⟦ !τ1 ↦ ⟦ !τ2 ↦ ∅, !B1 ⟧(!τ2 ↦ !n, !B2), !B3 ⟧ ], !B1 ⟧(!B2), !B3 ⟧
     when:
       - apply_in_abstract_subformations: false
-      - nf: '!b'
+      - nf: '!n'
     tests: []
```

Comment:
Can you add your example to the tests here and/or in `dataization.yaml` tests?

Reply:
added it here: [cb8121a](https://github.com/objectionary/normalizer/pull/445/commits/cb8121ac7f160781c9b1b6735f4c77e06d353db3)